packages/embark-ui/src/containers/EditorContainer.css

```diff
@@ -44,3 +47,62 @@
 .resize-handle-horizontal {
   height: 15px !important;
 }
+
+.dark-theme .tabs-container ul {
+  background-color: #212429
```

Comment:
Might be possible to convert this file to a `.scss`, import the variables file and use the color variable.
Or even easier, you could use the same class as before (probably a bootstrap class) so that the style is applied.

Reply:
Moved files to `.scss` and updated to colors in `variables/_colors.scss`.